src/storage/v3/name_id_mapper.hpp

```diff
@@ -47,6 +48,10 @@ class NameIdMapper final {
     return kUnmappedId;
   }
 
+  const std::string &IdToName(const LabelId label_id) const { return IdToName(label_id.AsInt()); }
+
+  const std::string &IdToName(const PropertyId property_id) const { return IdToName(property_id.AsInt()); }
+
```

Comment:
I think the `NameIdMapper` shouldn't now about label/property/type ids. It should remain a single `std::string<->uint64_t` bimap. I feel that it is questionable, but I am feeling that keeping it simple with simple responsibilities is better for the long term. If we need this functionality (I haven't seen it yet where do you use it), then a lightweight wrapper should solve the issue.